# Timestamped Bounding Boxes Array from Object Detection algo like YOLO
Header  header
bbox[] bboxes
